### PR TITLE
Pods 2.7.22

### PR DESCRIPTION
**Pull request by sc0ttkclark**

### CODE REVIEW

File: classes/PodsUI.php [hunk 1]
```diff
@@ -2911,6 +2911,10 @@ public function manage( $reorder = false ) {
 				foreach ( $filters as $k => $filter ) {
 					if ( isset( $this->pod->fields[ $filter ] ) ) {
 						$filter_field = $this->pod->fields[ $filter ];
+
+						if ( isset( $this->fields['manage'][ $filter ] ) && is_array( $this->fields['manage'][ $filter ] ) ) {
```
Comment (tr1b0t) [inline classes/PodsUI.php:2915]: :no_entry_sign: Line indented incorrectly; expected 7 tabs, found 6

<sub>hash: c710b082345dddc6b7fd3502bac1d853</sub>

File: classes/PodsUI.php [hunk 2]
```diff
@@ -2911,6 +2911,10 @@ public function manage( $reorder = false ) {
...
+							$filter_field = array_merge( $filter_field, $this->fields['manage'][ $filter ] );
```
Comment (tr1b0t) [inline classes/PodsUI.php:2916]: :no_entry_sign: Line indented incorrectly; expected at least 8 tabs, found 7

<sub>hash: 489c3ffdb7f0c4da82f82d38a1aaa2a4</sub>

File: classes/PodsUI.php [hunk 3]
```diff
@@ -2911,6 +2911,10 @@ public function manage( $reorder = false ) {
...
+						}
```
Comment (tr1b0t) [inline classes/PodsUI.php:2917]: :no_entry_sign: Line indented incorrectly; expected 7 tabs, found 6

<sub>hash: 3206b0277f6dab9c67af1b9e22e58800</sub>

File: classes/PodsUI.php [hunk 4]
```diff
@@ -2977,12 +2984,18 @@ public function manage( $reorder = false ) {
...
+										$filter_field = array_merge( $filter_field, $this->fields['manage'][ $filter ] );
+									}
 								} elseif ( isset( $this->fields['manage'][ $filter ] ) ) {
 									$filter_field = $this->fields['manage'][ $filter ];
 								} else {
 									continue;
 								}
-
+								?>
+								<span class="pods-form-ui-filter-container pods-form-ui-filter-container-<?php esc_attr_e( $filter ); ?>">
```
Comment (tr1b0t) [inline classes/PodsUI.php:2997]: :no_entry_sign: The $text arg must be a single string literal, not "$filter".

<sub>hash: 6b439d2c7e6bfb448a038c5ee1457f10</sub>

File: classes/PodsUI.php [hunk 5]
```diff
@@ -3002,13 +3015,37 @@ public function manage( $reorder = false ) {
 									<label for="pods-form-ui-filter-<?php esc_attr_e( $filter ); ?>_start">
 										<?php esc_html_e( $filter_field['label'] ); ?>
 									</label>
-									<?php echo PodsForm::field( 'filter_' . $filter . '_start', $start, $filter_field['type'], $filter_field ); ?>
+									<?php
+										// Prevent p div issues.
+										echo str_replace(
```
Comment (tr1b0t) [inline classes/PodsUI.php:3020]: :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'str_replace'.

<sub>hash: b965cc58400e44da320ed22a36ce98d1</sub>

File: classes/PodsUI.php [hunk 6]
```diff
@@ -3002,13 +3015,37 @@ public function manage( $reorder = false ) {
...
+											array(
+												'<div',
+												'</div>',
+											),
+											array(
+												'<span',
+												'</span>',
+											),
+											PodsForm::field( 'filter_' . $filter . '_start', $start, $filter_field['type'], $filter_field )
+										);
```
Comment (tr1b0t) [inline classes/PodsUI.php:3030]: :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: f459299046746637c307f15a0d30df7a</sub>

File: classes/PodsUI.php [hunk 7]
```diff
@@ -3002,13 +3015,37 @@ public function manage( $reorder = false ) {
...
+									?>
 
 									<label for="pods-form-ui-filter-<?php esc_attr_e( $filter ); ?>_end">
 										to
 									</label>
 								<?php
-									echo PodsForm::field( 'filter_' . $filter . '_end', $end, $filter_field['type'], $filter_field );
+									// Prevent p div issues.
+									echo str_replace(
```
Comment (tr1b0t) [inline classes/PodsUI.php:3038]: :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'str_replace'.

<sub>hash: 96c44044016971920d2d17d313c1e02a</sub>

File: classes/PodsUI.php [hunk 8]
```diff
@@ -3002,13 +3015,37 @@ public function manage( $reorder = false ) {
...
+										array(
+											'<div',
+											'</div>',
+										),
+										array(
+											'<span',
+											'</span>',
+										),
+										PodsForm::field( 'filter_' . $filter . '_end', $end, $filter_field['type'], $filter_field )
+									);
```
Comment (tr1b0t) [inline classes/PodsUI.php:3048]: :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 4c6dbb5f07d476df78ea4abab6dd6ccc</sub>

File: classes/PodsUI.php [hunk 9]
```diff
@@ -3030,7 +3068,18 @@ public function manage( $reorder = false ) {
 										<?php esc_html_e( $filter_field['label'] ); ?>
 									</label>
 								<?php
-									echo PodsForm::field( 'filter_' . $filter, $value, 'pick', $options );
+									// Prevent p div issues.
+									echo str_replace(
```
Comment (tr1b0t) [inline classes/PodsUI.php:3072]: :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'str_replace'.

<sub>hash: c26b5d4b2a37afe7b8c98baaf1a2c8d1</sub>

File: classes/PodsUI.php [hunk 10]
```diff
@@ -3030,7 +3068,18 @@ public function manage( $reorder = false ) {
...
+										array(
+											'<div',
+											'</div>',
+										),
+										array(
+											'<span',
+											'</span>',
+										),
+										PodsForm::field( 'filter_' . $filter, $value, 'pick', $options )
+									);
```
Comment (tr1b0t) [inline classes/PodsUI.php:3082]: :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 0f2606a7821d8f4215ca1e6c9cf6e42d</sub>

File: classes/PodsUI.php [hunk 11]
```diff
@@ -3058,7 +3108,18 @@ public function manage( $reorder = false ) {
 									<?php esc_html_e( $filter_field['label'] ); ?>
 									</label>
 									<?php
-									echo PodsForm::field( 'filter_' . $filter, $value, 'pick', $options );
+									// Prevent p div issues.
+									echo str_replace(
```
Comment (tr1b0t) [inline classes/PodsUI.php:3112]: :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'str_replace'.

<sub>hash: ceaa110293679c4373a00085c2fe1a4d</sub>

File: classes/PodsUI.php [hunk 12]
```diff
@@ -3058,7 +3108,18 @@ public function manage( $reorder = false ) {
...
+										array(
+											'<div',
+											'</div>',
+										),
+										array(
+											'<span',
+											'</span>',
+										),
+										PodsForm::field( 'filter_' . $filter, $value, 'pick', $options )
+									);
```
Comment (tr1b0t) [inline classes/PodsUI.php:3122]: :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: ffd3e4dc890d5b9e228d13da0a6011bb</sub>

File: classes/PodsUI.php [hunk 13]
```diff
@@ -3076,14 +3137,30 @@ public function manage( $reorder = false ) {
 										<?php esc_html_e( $filter_field['label'] ); ?>
 									</label>
 									<?php
-									echo PodsForm::field( 'filter_' . $filter, $value, 'text', $options );
+									// Prevent p div issues.
+									echo str_replace(
```
Comment (tr1b0t) [inline classes/PodsUI.php:3141]: :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'str_replace'.

<sub>hash: 9ca67974c5c79f609547b6987d791194</sub>

File: classes/PodsUI.php [hunk 14]
```diff
@@ -3076,14 +3137,30 @@ public function manage( $reorder = false ) {
...
+										array(
+											'<div',
+											'</div>',
+										),
+										array(
+											'<span',
+											'</span>',
+										),
+										PodsForm::field( 'filter_' . $filter, $value, 'text', $options )
+									);
```
Comment (tr1b0t) [inline classes/PodsUI.php:3151]: :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 73bcb3ff511c9e04b03843e964f3c9f8</sub>

File: classes/PodsUI.php [hunk 15]
```diff
@@ -3076,14 +3137,30 @@ public function manage( $reorder = false ) {
...
+									</span>
+								<?php
 							}//end foreach
 
 							if ( false !== $this->do_hook( 'filters_show_search', true ) ) {
 							?>
-								<label<?php echo ( empty( $this->filters ) ) ? ' class="screen-reader-text"' : ''; ?> for="page-search<?php esc_attr_e( $this->num ); ?>-input"><?php _e( 'Search', 'pods' ); ?>:</label>
-								<?php echo PodsForm::field( 'search' . $this->num, $this->search, 'text', array( 'attributes' => array( 'id' => 'page-search' . $this->num . '-input' ) ) ); ?>
+								<span class="pods-form-ui-filter-container pods-form-ui-filter-container-search">
+									<label<?php echo ( empty( $this->filters ) ) ? ' class="screen-reader-text"' : ''; ?> for="page-search<?php esc_attr_e( $this->num ); ?>-input"><?php _e( 'Search', 'pods' ); ?>:</label>
```
Comment (tr1b0t) [inline classes/PodsUI.php:3161]: * :no_entry_sign: The $text arg must be a single string literal, not "$this->num".
* :warning: Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e'

<sub>hash: a00544d27e4ccc9a6ebbf2ffcc98cfaa</sub>

File: classes/PodsUI.php [hunk 16]
```diff
@@ -3076,14 +3137,30 @@ public function manage( $reorder = false ) {
...
+									<?php echo PodsForm::field( 'search' . $this->num, $this->search, 'text', array( 'attributes' => array( 'id' => 'page-search' . $this->num . '-input' ) ) ); ?>
```
Comment (tr1b0t) [inline classes/PodsUI.php:3162]: * :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'PodsForm'.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 2bc65b9d5ea00006fd21386546d72dde</sub>

File: classes/PodsUI.php [hunk 17]
```diff
@@ -3593,10 +3681,36 @@ public function filters_popup() {
 								</label>
 
 								<span class="pods-ui-posts-filter<?php esc_attr_e( ( empty( $start ) && empty( $end ) ) ? ' pods-hidden' : '' ); ?>">
-								<?php echo PodsForm::field( 'filter_' . $filter . '_start', $start, $filter_field['type'], $filter_field ); ?>
+								<?php
+								// Prevent p div issues.
+								echo str_replace(
```
Comment (tr1b0t) [inline classes/PodsUI.php:3686]: :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'str_replace'.

<sub>hash: f461418dbb152b2f186cfa61cbaa23d1</sub>

File: classes/PodsUI.php [hunk 18]
```diff
@@ -3593,10 +3681,36 @@ public function filters_popup() {
...
+									array(
+										'<div',
+										'</div>',
+									),
+									array(
+										'<span',
+										'</span>',
+									),
+									PodsForm::field( 'filter_' . $filter . '_start', $start, $filter_field['type'], $filter_field )
+								);
```
Comment (tr1b0t) [inline classes/PodsUI.php:3696]: :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 749f5464b1029fe902dc5ffb6ee7ecbb</sub>

File: classes/PodsUI.php [hunk 19]
```diff
@@ -3593,10 +3681,36 @@ public function filters_popup() {
...
+								?>
 
 									<label for="pods-form-ui-filter-<?php esc_attr_e( $filter ); ?>_end">to</label>
-									<?php echo PodsForm::field( 'filter_' . $filter . '_end', $end, $filter_field['type'], $filter_field ); ?>
+									<?php
+									// Prevent p div issues.
+									echo str_replace(
```
Comment (tr1b0t) [inline classes/PodsUI.php:3702]: :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'str_replace'.

<sub>hash: c9d0727d69435e50561b100303987bfc</sub>

File: classes/PodsUI.php [hunk 20]
```diff
@@ -3593,10 +3681,36 @@ public function filters_popup() {
...
+										array(
+											'<div',
+											'</div>',
+										),
+										array(
+											'<span',
+											'</span>',
+										),
+										PodsForm::field( 'filter_' . $filter . '_end', $end, $filter_field['type'], $filter_field )
+									);
```
Comment (tr1b0t) [inline classes/PodsUI.php:3712]: :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 02590233b1901421d580d39e2423f832</sub>

File: classes/PodsUI.php [hunk 21]
```diff
@@ -3625,7 +3740,20 @@ public function filters_popup() {
 								</label>
 
 								<span class="pods-ui-posts-filter<?php esc_attr_e( strlen( $value ) < 1 ? ' pods-hidden' : '' ); ?>">
-								<?php echo PodsForm::field( 'filter_' . $filter, $value, 'pick', $options ); ?>
+								<?php
+								// Prevent p div issues.
+								echo str_replace(
```
Comment (tr1b0t) [inline classes/PodsUI.php:3745]: :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'str_replace'.

<sub>hash: 95bfe1d757607e620a5ece97bd8de027</sub>

File: classes/PodsUI.php [hunk 22]
```diff
@@ -3625,7 +3740,20 @@ public function filters_popup() {
...
+									array(
+										'<div',
+										'</div>',
+									),
+									array(
+										'<span',
+										'</span>',
+									),
+									PodsForm::field( 'filter_' . $filter, $value, 'pick', $options )
+								);
```
Comment (tr1b0t) [inline classes/PodsUI.php:3755]: :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 79c0d9447eae3464db239e8cb466986b</sub>

File: classes/PodsUI.php [hunk 23]
```diff
@@ -3660,7 +3789,20 @@ public function filters_popup() {
 								</label>
 
 								<span class="pods-ui-posts-filter<?php esc_attr_e( strlen( $value ) < 1 ? ' pods-hidden' : '' ); ?>">
-								<?php echo PodsForm::field( 'filter_' . $filter, $value, 'pick', $options ); ?>
+								<?php
+								// Prevent p div issues.
+								echo str_replace(
```
Comment (tr1b0t) [inline classes/PodsUI.php:3794]: :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'str_replace'.

<sub>hash: d0c07ff95cf9f5dd46bcaeca7f351633</sub>

File: classes/PodsUI.php [hunk 24]
```diff
@@ -3660,7 +3789,20 @@ public function filters_popup() {
...
+									array(
+										'<div',
+										'</div>',
+									),
+									array(
+										'<span',
+										'</span>',
+									),
+									PodsForm::field( 'filter_' . $filter, $value, 'pick', $options )
+								);
```
Comment (tr1b0t) [inline classes/PodsUI.php:3804]: :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 097979b6a31eae1de41c6fdbc65da539</sub>

File: classes/PodsUI.php [hunk 25]
```diff
@@ -3686,7 +3828,20 @@ public function filters_popup() {
 								</label>
 
 								<span class="pods-ui-posts-filter<?php esc_attr_e( empty( $value ) ? ' pods-hidden' : '' ); ?>">
-								<?php echo PodsForm::field( 'filter_' . $filter, $value, 'text', $options ); ?>
+								<?php
+								// Prevent p div issues.
+								echo str_replace(
```
Comment (tr1b0t) [inline classes/PodsUI.php:3833]: :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'str_replace'.

<sub>hash: 548087a89416638c1289bfc3c6c723d4</sub>

File: classes/PodsUI.php [hunk 26]
```diff
@@ -3686,7 +3828,20 @@ public function filters_popup() {
...
+									array(
+										'<div',
+										'</div>',
+									),
+									array(
+										'<span',
+										'</span>',
+									),
+									PodsForm::field( 'filter_' . $filter, $value, 'text', $options )
+								);
```
Comment (tr1b0t) [inline classes/PodsUI.php:3843]: :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 32b6ecd10d2efa23a6eda7a4da114855</sub>

File: ui/admin/form.php [hunk 27]
```diff
@@ -555,23 +551,18 @@
...
+
 									if ( ! $more && 1 == count( $groups ) ) {
 										$title = __( 'Fields', 'pods' );
 									} else {
 										$title = $group['label'];
 									}
 
 									/** This filter is documented in classes/PodsMeta.php */
-									echo apply_filters( 'pods_meta_default_box_title', $title, $pod, $fields, $pod->api->pod_data['type'], $pod->pod );
+									$title = apply_filters( 'pods_meta_default_box_title', $title, $pod, $fields, $pod->api->pod_data['type'], $pod->pod );
```
Comment (tr1b0t) [inline ui/admin/form.php:562]: :no_entry_sign: Overriding WordPress globals is prohibited. Found assignment to $title

<sub>hash: 70fc25a3711d14f5817cbd31799d7a8b</sub>

File: ui/admin/postbox-header.php [hunk 28]
```diff
@@ -0,0 +1,41 @@
...
+
+<?php if ( version_compare( $GLOBALS['wp_version'], '5.5', '>=' ) ) : ?>
+	<div class="postbox-header">
+		<h2 class="hndle">
+			<span><?php echo esc_html( $title ); ?></span>
+		</h2>
+		<div class="handle-actions hide-if-no-js">
+			<?php
+			/* We need to build support for this later.
+			<button type="button" class="handle-order-higher" aria-disabled="false" aria-describedby="<?php echo esc_attr( $box_id ); ?>-handle-order-higher-description">
```
Comment (tr1b0t) [inline ui/admin/postbox-header.php:17]: :no_entry_sign: Block comment text must start on a new line

<sub>hash: 656a440292de62c54e0cf19e88108204</sub>

File: ui/admin/postbox-header.php [hunk 29]
```diff
@@ -0,0 +1,41 @@
...
+			</button>
+		</div>
+	</div>
+<?php else : ?>
+	<div class="handlediv" title="<?php esc_attr_e( 'Click to toggle', 'pods' ); ?>"><br></div>
+	<h3 class="hndle">
+		<span><?php echo esc_html( $title ); ?></span>
+	</h3>
+<?php endif; ?>
+
```
Comment (tr1b0t) [inline ui/admin/postbox-header.php:41]: :no_entry_sign: Additional whitespace found at end of file

<sub>hash: 0d8232ea4551bb6ba001e05587a92b30</sub>

File: ui/admin/view.php [hunk 30]
```diff
@@ -182,24 +180,18 @@
 				}//end if
 
 				if ( 0 < count( $fields ) ) {
+					if ( $more ) {
+						$title = __( 'More Fields', 'pods' );
```
Comment (tr1b0t) [inline ui/admin/view.php:184]: :no_entry_sign: Overriding WordPress globals is prohibited. Found assignment to $title

<sub>hash: 50c8b86e26838c1b8fb139b0eb625bc8</sub>

File: ui/admin/view.php [hunk 31]
```diff
@@ -182,24 +180,18 @@
...
+					} else {
+						$title = __( 'Fields', 'pods' );
```
Comment (tr1b0t) [inline ui/admin/view.php:186]: :no_entry_sign: Overriding WordPress globals is prohibited. Found assignment to $title

<sub>hash: 9e665f2698124fd693173ea75e6e5c2f</sub>

File: ui/admin/view.php [hunk 32]
```diff
@@ -182,24 +180,18 @@
...
+					}
+
+					/** This filter is documented in classes/PodsMeta.php */
+					$title = apply_filters( 'pods_meta_default_box_title', $title, $pod, $fields, $pod->api->pod_data['type'], $pod->pod );
```
Comment (tr1b0t) [inline ui/admin/view.php:190]: :no_entry_sign: Overriding WordPress globals is prohibited. Found assignment to $title

<sub>hash: f03c340e77ab6edf9e2cc1bc991bce0a</sub>

File: classes/PodsUI.php (hunk 15 above)
Comment (sc0ttkclark) [reply classes/PodsUI.php:3161]:
```suggestion
									<label<?php echo ( empty( $this->filters ) ) ? ' class="screen-reader-text"' : ''; ?> for="page-search<?php echo esc_attr( $this->num ); ?>-input"><?php _e( 'Search', 'pods' ); ?>:</label>
```

File: classes/PodsUI.php (hunk 4 above)
Comment (sc0ttkclark) [reply classes/PodsUI.php:2997]:
```suggestion
								<span class="pods-form-ui-filter-container pods-form-ui-filter-container-<?php echo esc_attr( $filter ); ?>">
```

**Pull request merged**